h/subscribers.py

```diff
@@ -27,6 +27,7 @@ def add_renderer_globals(event):
     if "h.sentry_dsn_frontend" in request.registry.settings:
         event["frontend_settings"]["sentry"] = {
             "dsn": request.registry.settings["h.sentry_dsn_frontend"],
+            "environment": request.registry.settings["h.sentry_environment"],
```

Comment:
`h.sentry_environment` is currently always set. It takes its value from the `SENTRY_ENVIRONMENT` env var with a default of `dev`.

Reply:
We have (in h-prod):

`SENTRY_ENVIRONMENT`

and 

`ENV`

both with value a value of `prod` :+1: 


We might want to look at combine those at some point, outside of scope here.